SceneRefValidatorOnSave.cs

```diff
@@ -0,0 +1,51 @@
+#if UNITY_EDITOR
+
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace KBCore.Refs
+{
+    [InitializeOnLoad]
+    public static class SceneRefValidatorOnSave
+    {
+        private const string PrefsKey = "KBCore/ValidateRefsOnSave";
+        private const string MenuItemText = "Tools/KBCore/Validate Refs on Save";
+        
+        public static bool ValidateRefsOnSave
+        {
+            get => EditorPrefs.GetBool(PrefsKey, true);
```

Comment:
LGTM but I'd propose this be kept false by default, as it's going to be very slow on larger projects that are already using the library

Reply:
All right, I changed it